test/middleware/spec.back-link.js

```diff
@@ -134,24 +139,25 @@ describe('Back Links', function () {
 
     it('strips baseUrl before checking whitelist', function () {
         req.get.withArgs('referrer').returns('http://example.com/base/whitelist');
-        req.sessionModel.set('steps', ['/step1', '/step2']);
         req.baseUrl = '/base';
+        req.sessionModel.set('steps', ['/step1', '/step2']);
         controller.options.backLinks = ['whitelist'];
         backLinks('/step3', controller, steps)(req, res, next);
         res.locals.backLink.should.equal('whitelist');
     });
 
     it('supports absolute paths in whitelist', function () {
         req.get.withArgs('referrer').returns('http://example.com/whitelist');
+        req.baseUrl = '/whitelist';
         req.sessionModel.set('steps', ['/step1', '/step2']);
-        req.baseUrl = '/base';
```

Comment:
Changing the baseUrl doesn't affect the outcome of the test, it can also be removed, but keeping it in shows that it doesn't affect the back link, IMO.